test/services/cache/ExportsDeclarationBuilder.scala

```diff
@@ -146,9 +146,11 @@ trait ExportsDeclarationBuilder {
 
   private def uuid: String = UUID.randomUUID().toString
 
-  def withEntryIntoDeclarantsRecords(isEidr: String = YesNoAnswers.yes): ExportsDeclarationModifier =
+  def withEIDR(isEidr: String = YesNoAnswers.yes): ExportsDeclarationModifier =
     cache => cache.copy(parties = cache.parties.copy(isEntryIntoDeclarantsRecords = Some(YesNoAnswer(isEidr))))
 
+  def withEntryIntoDeclarantsRecords(isEidr: String = YesNoAnswers.yes): ExportsDeclarationModifier = withEIDR(isEidr)
```

Comment:
Why have two methods with different names doing the same thing?

Reply:
Same answer